### PR TITLE
Evelina , readme file

### PR DESCRIPTION
**Pull request by EvelinaTemnolonskyte**
Nice team , nice work

### CODE REVIEW

**Comment by arminasbrazenas** [comment]
Lets use README.md for our documentation, There already is a file in the repository root. Here is an example for markdown in VS Code: https://code.visualstudio.com/docs/languages/markdown

**Comment by arminasbrazenas** [comment]
Pull request name should be descriptive. 1st and 2nd tips from [this](https://blog.montrealanalytics.com/4-tips-for-effective-pull-request-naming-f60793998f04) tutorial seem to elaborate it.

**Comment by arminasbrazenas** [comment]
Also, for future reference, name the branch according to the feature that you are implementing or even have it under your own directory: e.g. evelina/update-readme

**Comment by EvelinaTemnolonskyte** [comment]
Thank you!

File: studyzen
Comment (vestacic) [inline studyzen:1]: I believe no modules should be added yet

**Review by vestacic — CHANGES_REQUESTED** [review]

**Comment by EvelinaTemnolonskyte** [comment]
Have solved this in new pull request

**Comment by arminasbrazenas** [comment]
> Have solved this in new pull request

Nice, next time there is no need to create a new pull request. You can do the following:

1. `git log`
2. find id of the commit that you want to revert your changes to
3. `git reset {commitId}` (be aware that there is soft and hard reset)
4. make changes
5. `git push` (might need to use `--force` flag)

**Pull request closed without merging**